### ISSUE TITLE
Recent Folders tab - keyboard support

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#562]
Looks like the Recent folders tab cannot be navigated to with the keyboard. 

- Expected behavior: the Tab key should go from the QF buttons (QuickFolderf label with tools, reading list, quickJump to Recent Folders tab and then to the list of tabs (where we can navigate with arrow buttons)
- observed behavior: Recent folder tab is skiped
- Workaround: use the quickJump button and the shortcut key `=` to show recent folders

**Comment by RealRaven2000** [comment]
The problem was caused by misspelling tabindex with an uppercase I.

here is a quick fix:

[QuickFolders-mx-6.10.2pre1.zip](https://github.com/user-attachments/files/19398304/QuickFolders-mx-6.10.2pre1.zip)

----

To test this version, download the zip file and then drag it into Thunderbird Add-ons Manager (without extracting)
for users of screenreader you can also install this via the tools button after the "Find more addons" search text. activate the button with Enter then chose the 3rd menu entry "install addon from file...". Then select the zip file to install.

**Comment by RealRaven2000** [comment]
Found another bug that created a nonsensical top item on the recent folders list. Just fixed it and will roll it into maintenance release 6.10.2.

**Comment by RealRaven2000** [comment]
completed in 6.10.3 released on 24/03/2025
